test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -1138,6 +1138,90 @@ static void test_out_ipv4_udp_esp_null_sha256(void)
 	ipsec_sa_destroy(sa);
 }
 
+static void test_out_ipv4_esp_aes_gcm_128_sa_info(void)
```

Comment:
How about naming this simply test_sa_info? I suppose the algorithm chosen for this test is not an essential and you are not planning to add a bunch of similar tests with different algorithms.

This test does not test the dynamic state info like recent sequence number.

Reply:
> 
> 
> How about naming this simply test_sa_info? I suppose the algorithm chosen for this test is not an essential and you are not planning to add a bunch of similar tests with different algorithms.

test_sa_info sounds good. Yes, there is no importance on the algorithm chosen as long as the algorithm is supported on the platform.

> 
> This test does not test the dynamic state info like recent sequence number.

We can do something like, send one packet and then do an sa_info call to see that the seq_num was incremented in the same test.